packages/docs/pages/ui-onboarding/index.md

```diff
@@ -1,3 +1,94 @@
+  - Remove the deploy\_key.enc from the .travis folder in the source repo.
+  - [Example](https://github.com/RedHatInsights/insights-remediations-frontend/tree/master/.travis)
+3. **Generate the new key** (in the source repo)
+  - ssh-keygen -t rsa -b 4096
+    Save the key without a passphrase
+  - Save the key as deploy\_key in {source-repo}/.travis/deploy\_key
+    i. You should get a deploy\_key and deploy\_key.pub
+
+4. **Add the new deploy key to the build repo** (build repo)
+  - Copy the contents of the public key generated in step 2 ( **deploy\_key.pub** ) to the build repo under Settings → Deploy Keys and check the box for allowing write access. You can name this &quot;Travis CI&quot;.
+  - [Example](https://github.com/RedHatInsights/insights-remediations-frontend-build/settings/keys)
+5. **Encrypt the deploy key with Travis CLI** (source repo)
+  - Make sure you have [Travis CI CLI tools](https://github.com/travis-ci/travis.rb)
+    i. [Installation](https://github.com/travis-ci/travis.rb#installation) ** **
+  - [Add a personal access token](https://github.com/settings/tokens) in github if you don&#39;t have one already
+  - Login to Travis CLI with the --com flag with your github token
+    i. travis login --com --github-token {token}
+  - Verify that you are inside of your development repo and then encrypt the key with Travis CLI
+    i. travis encrypt-file .travis/deploy\_key --com
+  - You may be prompted to add an openssl command, **do not add the openssl command to the .travis file.**
+  - After you encrypt the file, they automatically get added to travis
+6. **Verify**
+  - You should now have a deploy\_key.enc file
+  - Delete deploy\_key and deploy\_key.pub
+7. **Commit**
+  - Commit your deploy\_key.enc file _**(DO NOT COMMIT THE PRIVATE KEY)**_
+  - Open a PR against master branch
+8. **Verify that the Travis env variables have been added**
+  - [https://app.travis-ci.com/github/RedHatInsights/{source-repo}/settings](https://app.travis-ci.com/github/RedHatInsights/%7Bsource-repo%7D/settings)
+  - The hash for the key and iv should match the hash you see in the logs after you run the encryption in Step 7
+9. **Confirm** the build completed successfully on Travis-CI and verify that the files were pushed to your build repo.
+
+
+# Deploying Your App
+
+After the dev repo is set up, you can deploy your app using various branches. By default, a branch corresponds to an environment. Example)
+  - branch: ci-beta will push to the ci-beta environment: ci.console.redhat.com
+
+The naming convention follows env-{beta/stable}
+
+# Custom Releases
+
+Many teams want alignment between environments on the platform, so we allow a .custom\_release.sh file inside of your .travis directory in your dev repo.
+
+You can set up any push logic you&#39;d like to in this repo. Many times, teams will align ci/qa together, so an example of that would be: ![](RackMultipart20220401-4-1ogbjj7_html_3d8d7472659d9099.png)
```

Comment:
Do we have the PNG?

Reply:
I believe it's this
![image](https://user-images.githubusercontent.com/56621323/161696687-09080ae7-0c93-4fc5-b708-24c48089422b.png)

Reply:
@arivepr but it's missing in the changelog. Instead of an image I'd rather pasted `yaml` partial. That way consumers can copy+paste it to their config.

Reply:
Yup if it's just a piece of code, can you directly insert it into the docs?